src/server/mod.rs

```diff
@@ -534,10 +534,7 @@ impl LanguageServer for LspServer {
                                             Ok(composition.clone())
                                         }
                                     }
-                                    Err(_) => {
-                                        log::error!("Found composition but did not find ast for key: {}", key);
-                                        Err(LspClientCommand::CompositionNotFound)
-                                    }
+                                    Err(_) => Err(LspClientCommand::CompositionNotFound),
```

Comment:
This is triggered when we select "New Script" in the UI. So I didn't feel like that use case should output to the browser console.